Feat(spv 1493) update js client to a new version with split admin and user client

cain4chain opened the pull request:
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

cain4chain committed:
feat(SPV-1493):update js-client to a new version with split admin and user client

chore(js-client): update js client version
chore(createClient): Define extended client types for admin and user API
chore(apis): Update all APIs to use new version for admin and user client from the new js-client library version 1.0.0-beta.32

cain4chain committed:
feat(SPV-1493): use tanstack store and react hooks for admin and user API

chore(clientStore): update client store using tanstack library for state management
chore(clientStore): use react hooks for user and admin API
chore(apis): update all API interactive files using the new client store version

cain4chain committed:
feat(SPV-1493) : add client store for admin and user seperation

feat(clientStore) : integrate client store into remaining endpoints
feat(lint) : update lint issues

cain4chain committed:
Merge branch 'main' into feat(SPV-1493)-update-js-client-to-a-new-version-with-split-admin-and-user-client

cain4chain committed:
feat(SPV-1493) : Update conflicts

chore(paymail_query) : update paymail query utility with new js client format

cain4chain committed:
Merge branch 'main' into feat(SPV-1493)-update-js-client-to-a-new-version-with-split-admin-and-user-client

cain4chain committed:
chore(update j client): js client version update

chore(update j client): js client version update to latest

cain4chain committed:
chore(install dep updates) : yarn lock updated

chore(install dep updates) : yarn lock updated to latest

cain4chain committed:
feat(SPV-1493) :  Remove delete dialog for user access

chore(paymailsTabContent) : remove delete dialog access for user
fix(paymailDeleteDialog) : change paymail to id for delete paymail

cain4chain committed:
feat(SPV-1493) : Update styling

chore(prettier) : run prettier formatting

cain4chain committed:
feat(SPV-1493) - update admin status check

chore(clientStore) : add function for admin status check

cain4chain committed:
Merge branch 'main' into feat(SPV-1493)-update-js-client-to-a-new-version-with-split-admin-and-user-client

wregulski commented on package.json:
```diff
@@ -23,7 +23,7 @@
     "@4chain-ag/react-configuration": "^1.0.4",
     "@bsv/sdk": "^1.3.12",
-    "@bsv/spv-wallet-js-client": "^1.0.0-beta.30",
+    "@bsv/spv-wallet-js-client": "^1.0.0-beta.33",
```
It should be:
`"@bsv/spv-wallet-js-client": "^1.0.0-beta.36",`
if we could try to bump to newest version.
It shouldn't be so much changes.

cain4chain committed:
chore(package.json) - version update

chore(package.json) : update to latest version of SPV- JS client (1.0.0-beta.36)

cain4chain committed:
chore(yarn.lock) - update latest

chore(yarn.lock) - install latest packages
